### ISSUE TITLE
document

### ISSUE THREAD

**Issue by Runfa-Zhang** [tensordiffeq/TensorDiffEq#29]
I'm very interested in your project, so do you have a demonstration document about TensorDiffEq?

**Comment by levimcclenny** [comment]
Hey Runfa - check out the docs at tensordiffeq.io and we have a short paper out (that needs some revision now) here - https://arxiv.org/abs/2103.16034

**Comment by levimcclenny** [comment]
Additionally the scripts in the examples/ folder will run out-of-the-box on a CPU or GPU machine

**Comment by Runfa-Zhang** [comment]
> Hey Runfa - check out the docs at tensordiffeq.io and we have a short paper out (that needs some revision now) here - https://arxiv.org/abs/2103.16034

Thanks very much for your help！

**Comment by Runfa-Zhang** [comment]
Can SAPINNs get a predict analytical solutions？

In fact, returning the expression of the neural network model can answer my problem.

so how to get or which code？in SA PINNs？

Thanks again for your time.

**Comment by Runfa-Zhang** [comment]
> Hey Runfa, 
> 
> PINNs, like all NNs, act as function approximators. So getting a closed-form symbolic analytical solution from a NN solution, or a PINN solution is not (to my knowledge) a bridge that has been crossed. The premise is that the solutions to systems of PDEs  that don't already have analytical solutions are incredibly complex to derive in closed-form, and therefore an approximation network is a more efficient representation and also simpler to learn since it can be trained via a loss function and gradient descent. 
> 
> In the case of TDQ, the NN solution is an approximation network and can be evaluated and plotted, examples of which we have in the `examples/` folder. We don't return analytical solutions, and if you find software that does please let me know!
# Many thanks for your kind reply！
